File: api/java/toolgood.textfilter.api/src/main/java/toolgood/textfilter/api/Datas/Keywords/KeywordListResult.java
package toolgood.textfilter.api.Datas.Keywords;

import java.util.List;

public class KeywordListResult {
    /**
     * 返回码：0) 成功，1) 失败
     */
    public int code;
    /**
     * 返回码详情描述
     */
    public String message;

    /**
     * 总个数
     */
    public Integer total;

    /**
     * 自定义敏感词列表
     */
    public List<KeywordItem> data;
}

File: api/python/Datas/Keywords/KeywordListResult.py
#!/usr/bin/env python
# -*- coding:utf-8 -*-
# KeywordItem.py
# 2021, Lin Zhijun, https://github.com/toolgood/ToolGood.TextFilter.Api
# MIT Licensed 
 
__all__ = ['ToolGood.TextFilter.KeywordItem']

class KeywordListResult():
    '返回码：0) 成功，1) 失败'
    code=0
    '返回码详情描述'
    message=""
    '总个数'
    total=0
    '自定义敏感词列表'
    data=[]
 

 